.github/workflows/pfm-pr-merged.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark Asana task complete on merge
+
+on:
+    pull_request:
+        types: [labeled, closed]
+
+jobs:
+    add-pr-merged-comment:
+        if: contains(join(github.event.pull_request.labels.*.name, ','), 'SBT-PFM') && (github.event.action == 'closed') && (github.event.pull_request.merged == true)
```

Comment:
Now that I see the whole thing, I think you can simplify and remove `labeled` from `types` above. Then you can remove `(github.event.action == 'closed')` here. The same applies to `pfm-pr-opened.yml` and `pfm-pr-review-notifications.yml`.